Description:  Add libraries to LIBADD in Makefile.am 
 Adding libraries to LIBADD in Makefile.am resolves the many 
 not found in and library errors generated by dpkg-shlibsdeps. 
Author: Scarlett Moore <sgclark@kde.org> 
Last-Update: 2018-09-29  

--- a/Makefile.am
+++ b/Makefile.am
@@ -28,13 +28,13 @@
 	util.h fs.h
 libsquashfuse_la_CPPFLAGS = $(ZLIB_CPPFLAGS) $(XZ_CPPFLAGS) $(LZO_CPPFLAGS) \
 	$(LZ4_CPPFLAGS) $(ZSTD_CPPFLAGS)
-libsquashfuse_la_LIBADD =
+libsquashfuse_la_LIBADD = $(COMPRESSION_LIBS)
 
 # Helper for FUSE clients: libfuseprivate
 libfuseprivate_la_SOURCES = fuseprivate.c nonstd-makedev.c nonstd-enoattr.c \
 	fuseprivate.h
 libfuseprivate_la_CPPFLAGS = $(FUSE_CPPFLAGS)
-libfuseprivate_la_LIBADD =
+libfuseprivate_la_LIBADD = libsquashfuse.la $(FUSE_LIBS)
 
 # High-level squashfuse
 if SQ_WANT_HIGHLEVEL
